recipes/artery-font-format/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0":
+    url: "https://github.com/Chlumsky/artery-font-format/archive/refs/heads/master.zip"
```

Comment:
Master is a changing target, download a specific commit

Reply:
I changed to point to the current master commit, and will update it if a version tag is created.